numpy/polynomial/_polybase.py

```diff
@@ -67,6 +68,37 @@ class ABCPolyBase(abc.ABC):
     # Limit runaway size. T_n^m has degree n*m
     maxpower = 100
 
+    # Unicode character mappings for improved __str__
+    _superscript_mapping = str.maketrans({
+        "0" : "⁰",
```

Comment:
You have a space before the `:` in these dictionary items.  I always thought that was against the PEP 8 recommendations, but after taking another look at the document, I see that it never explicitly says that. However, the examples in the PEP 8 document don't use a space before `:` when creating a dictionary item, and `flake8` will complain about it, so I recommend not using a space.   But I won't insist, and I don't know if anyone else will mind, so your call...

Reply:
I'm unaware of what PEP8 dictates here either, but if it's getting caught by linters then I agree it should probably be changed - thanks!